ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/console/CPPOutputCustomizer.java

```diff
@@ -0,0 +1,184 @@
+/*******************************************************************************
```

Comment:
@vparfonov In order to realize the second (moving customizers to according plug-ins and adding them to a console basing on active workspace configuration - did you mention it as a 'second variant'?), I'd like to push this PR to close the issue and have the code pushed, and then start a new one like "Move output console customizers into according plugins and do output consoles' customizing based on workspace runtimes configuration".
I'd like to work on such an issue, but for sure I'll need some support/consultations on how can I create/initialize customizers from their plug-ins, how can I gather the output customizers that are required to be working for a given workspace config or how can I create a preferences for allowing a user the manual customizers configuration on a workspace (if we decide to allow manual configuration) and so on.